.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,8 @@
+blank_issues_enabled: false
```

Comment:
I would not block this option, because personally I sometimes file issues for refactoring, creating a tool or some raw ideas.

Reply:
it seems you get it wrong

> The blank_issues_enabled option allows you to show or hide the Create a blank issue choice when users select New issue in your repository. 
https://github.blog/changelog/2019-10-28-new-issue-template-configuration-options/

I think it is better to hide "Create a blank issue".

Reply:
Seems no. I want to have ability to create an issue, which is not a bug report and not a feature request. And I would want it being an external man.

Reply:
I believe it happens rarely and it is better to choose issue with template and erase text.
I'm afraid that with available option "Create a blank issue" users will prefer it than issues with templates, because filling all required fields in templates requires more strength.

Reply:
I guess an external user will be unable to add or removed labels.

Reply:
Well, decided to keep option enabled.

Reply:
Thanks! Let's return here later, if we'll see any negative impact.